app/scripts/controllers/anchorScroll.ctrl.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+angular.module('yourStlCourts').controller('AnchorScrollCtrl', function ($anchorScroll) {
```

Comment:
This is an odd setup. I get that you are trying to use the same controller on two pages but it's kind of limiting to do it that way for page controllers. It's also generally not good practice to put services on $scope (or ctrl with "controller as") because it's exposing a service directly to the view which is crossing two layers of the software and makes it less flexible.